.github/workflows/config.yml

```diff
@@ -0,0 +1,35 @@
+# This workflow will do a clean installation of node dependencies, cache/restore them, build the source code and run tests across different versions of node
+# For more information see: https://docs.github.com/en/actions/automating-builds-and-tests/building-and-testing-nodejs
+
+name: RideOn-frontend
+
+on:
+  push:
+    branches: [ "main", "*", "*/*", "**" ]
+  pull_request:
+    branches: [ "main" ]
+
+jobs:
+  build-and-test:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        node-version: [16.x]
+        # See supported Node.js release schedule at https://nodejs.org/en/about/releases/
+
+    steps:
+    - name: Check out repository
+      uses: actions/checkout@v3
+    - name: Use Node.js ${{ matrix.node-version }}
+      uses: actions/setup-node@v3
+      with:
+        node-version: ${{ matrix.node-version }}
+        cache: 'npm'
+    - name: Installing dependencies
+      run: npm ci
+    - name: Running build
```

Comment:
linting and tests are not dependent on build. So can we run this build script after running unit tests

Reply:
Any reason for shifting test before build? What I have seen,  Build ---> Lint ----> Test is the most common practice which is widely used.

Reply:
We have separated build and test job, both are running parallelly